source/kraken/worker.cpp

```diff
@@ -230,32 +230,33 @@ pbnavitia::Response Worker::next_stop_times(const pbnavitia::NextStopTimeRequest
 
     bt::ptime from_datetime = bt::from_time_t(request.from_datetime());
     bt::ptime until_datetime = bt::from_time_t(request.until_datetime());
+    bt::ptime current_datetime = bt::from_time_t(request._current_datetime());
     try {
         switch(api) {
         case pbnavitia::NEXT_DEPARTURES:
             return timetables::next_departures(request.departure_filter(),
                     forbidden_uri, from_datetime,
                     request.duration(), request.nb_stoptimes(), request.depth(),
-                    type::AccessibiliteParams(), *data, false, request.count(),
-                    request.start_page(), request.show_codes());
+                    type::AccessibiliteParams(), *data, true, request.count(),
```

Comment:
don't we want to have an API parameter to display or not the disruptions for those query ?

Not sure it's always relevant.